### ISSUE TITLE
Section 508 502.3.1: (button 'Page up') The BoundingRectangle property must represent an area of at least 25 pixels.

### ISSUE THREAD

**Issue by tgugler** [wixtoolset/issues#6362]
If this issue is a bug:

* Which version of WiX are you building with?

> 3.14.0.1119

* Which version of Visual Studio are you building with (if any)?

> Visual Studio 2017

* Which version of the WiX Toolset Visual Studio Extension are you building with (if any)?

> 1.0.0.4

* Which version of .NET are you building with?

> Not applicable

* If the problem occurs when installing your packages built with WiX, what is the version of Windows the package is running on?

> Not applicable

* Describe the problem and the steps to reproduce it.

User Experience:
If, The BoundingRectangle property doesn't represent an area of at least 25 pixels, then the AT users  might get affected while navigating on the screen controls and unable to perform its function and access the desired control properly.

Pre Requisite:
Download exe file from https://www.microsoft.com/en-us/download/details.aspx?id=54688

Repro Steps:
Launch the "NPS Extension For Azure MFA" setup file.
Navigate all the controls of "MICROSOFT SOFTWARE LICENSE TERMS" popup window.
Run Accessibility Insights for windows and verify the issue.

Actual Result :
The BoundingRectangle property doesn't represent an area of at least 25 pixels.

Expected Result:
The BoundingRectangle property must represent an area of at least 25 pixels.

![1284869-pic](https://user-images.githubusercontent.com/12263599/108532426-b144dc00-728c-11eb-962e-de396afe801d.png)

**Comment by BMurri** [comment]
The scrollbar is owned by the document control. Same comment as I made on the other issue #6361.
